Sources/MenuAnimator.swift

```diff
@@ -179,7 +191,10 @@ class MenuInteractiveTransition: NSObject, UIViewControllerInteractiveTransition
     //MARK: - Private methods
     //
     private func createSnapshotView(from: UIView) -> UIView {
-        let snapshotView = from.snapshotView(afterScreenUpdates: true)!
+        let snapshotViewOptional = from.snapshotView(afterScreenUpdates: true)
+        guard let snapshotView = snapshotViewOptional else {
+            fatalError("Invalid snapshot view")
```

Comment:
When does `snapshotView` return a `nil`?  I mean is it really fatal situation (and the app should be terminated here)?